CMSgov/easi-app
# [NOREF] - Adjust intake summary statuses

# NOREF

## Changes and Description

- System intakes were showing statuses based on step for admins/requesters when an intake had a decision, was re-opened,  progressed to an earlier step, and then closed without confirming a decision. This change computes a closed status for those intakes.
- Tests cases were added for the admin and requester resolvers.

## How to test this change

Run the backend resolver tests and/or:
1. Make an intake.
2. Submit it.
3. Issue a decision.
4. Re-open it.
5. Progress it to an earlier step.
6. Re-close it without issuing/reissuing a decision.
7. Verify the status fields in the table views for requesters/admins.

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
